### ISSUE TITLE
Display real, non-friendly channel name in Things->Channels

### ISSUE THREAD

**Issue by gersilex** [openhab/org.openhab.ui.habmin#251]
As a **User**
I want to **see the real channel name in the channels overview of a thing**
so I can **read these information to link my items to this channel in textual configuration**

Right now the needed information can be found by clicking on the "Create new Item for channel" button in the respective channel line: 
![image](https://cloud.githubusercontent.com/assets/2018039/21750755/e807b5c8-d5ba-11e6-9aef-16b44fd5cd5f.png)

HABmin recommends a link name that consists of the thing and the channel, seperated by underscores. I can use this information to write the channel link into my items file:
![image](https://cloud.githubusercontent.com/assets/2018039/21750764/0afbe9fa-d5bb-11e6-84f1-0e3f9dcdc1e2.png)
![image](https://cloud.githubusercontent.com/assets/2018039/21750768/2575d912-d5bb-11e6-94aa-c16edb9f6742.png)

It would be much more comfortable to just see the channel name after the friendly name or onmouseover:
![image](https://cloud.githubusercontent.com/assets/2018039/21750792/b32cc810-d5bb-11e6-9488-4f1a8c5d159f.png)

A/C:
- Channel name can be found without additional clicks

Thank you very much.

**Comment by cdjackson** [comment]
I don't think you really want the channel name do you - the image you show is showing the item name - not the channel.

The best place to get the item name information is in the channels page where items are linked to channels -:

![image](https://cloud.githubusercontent.com/assets/1041475/21750963/2b91a612-d5e5-11e6-9f99-d5a7170a326c.png)
![image](https://cloud.githubusercontent.com/assets/1041475/21750966/38f96542-d5e5-11e6-8a0a-1ab2cd9ce9b0.png)

If you hold the mouse over the item name, it will bring up the little 'copy icon' (see the image above) and if you click on this it will copy the name directly to the clipboard which sounds like it's what you effectively want to do?

**Comment by gersilex** [comment]
I use HABmin for discovery and thing creation, because it's perfect for this and I need a place to configre my z-wave devices. Great thing btw.

But I use the textual configuration of openhab2 for everything else possible. 

In this case, I have already set up the things using HABmin:
![image](https://cloud.githubusercontent.com/assets/2018039/21751038/a000be86-d5c0-11e6-8442-eb875e52c0ce.png)

Now I go into my items file and link the item to the channel for being able to switch the power outlet thing:
![image](https://cloud.githubusercontent.com/assets/2018039/21751054/e00d9422-d5c0-11e6-99c0-d5a88bf4182c.png)

Here you can see that I have to find the channel name to be able to insert it right where the cursor is.

I hope I could make it somewhat clear :smiley:

**Comment by cdjackson** [comment]
Ok - understood. I thought when you showed the item edit box that you were after the items.

Anyway - I’ll take a look at adding this.

**Comment by gersilex** [comment]
Oh I just saw how PaperUI does displays it:
![image](https://cloud.githubusercontent.com/assets/2018039/21752615/324bd922-d5dc-11e6-97ff-a0ed5bd7fbca.png)

That is even better and perfect for copy&pasting. What do you think @cdjackson ?
